org.dita-semia.postprocessing.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "name": "org.dita-semia.postprocessing",
+    "description": "Set pdf filename depending on content (e.g. metadata) and zipping the result.",
+    "keywords": ["zip", "filename"],
+    "homepage": "https://github.com/dita-semia/org.dita-semia.postprocessing",
+    "vers": "0.1",
+    "license": "Apache-2.0",
+    "deps": [
+      {
+        "name": "org.dita.base",
+        "req": "2.4"
```

Comment:
This should be `2.4.0` (or `>=2.4.0` if you mean works on DITA-OT 2.4.0 or newer)

Reply:
Done!